src/umockdev-ioctl.vala

```diff
@@ -65,6 +65,30 @@ public class IoctlData {
         this.stream = stream;
     }
 
+    /**
+     * umockdev_ioctl_data_ref:
+     * @self: A #UMockdevIoctlData
+     *
+     * Deprecated, same as g_object_ref().
```

Comment:
Why do we need this? Introducing an already deprecated API feels a bit strange, and these are not being used from anywhere.

Reply:
Because vala defined these for the non-GObject based type. With GObject as the base class, we obviously don't need a custom ref/unref anymore, but need to add the methods to remain ABI compatible.

The only existing user (outside of umockdev itself) is the libusb test.